src/index.jsx

```diff
@@ -1,7 +1,11 @@
 import { createRoot } from 'react-dom/client';
 import { App } from './App';
-
+import { BrowserRouter as Router } from 'react-router-dom';
 import './index.css';
 
 const root = createRoot(document.getElementById('root'));
-root.render(<App />);
+root.render(
```

Comment:
my comments here do not have much practical value, but I wanted to express that I really like the router refactoring